src/test/java/com/google/cloud/tools/appengine/api/deploy/AppYamlProjectStagingTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google Inc.
+ * Copyright 2016 Google LLC.
```

Comment:
I looked at our internal doc, and there is no period. Makes sense LLC is not a shortened abbreviation but an acronym.

Reply:
From what I found, it recommends the period. IM'd you the link.